bitnami/rabbitmq/templates/config-secret.yaml

```diff
@@ -12,6 +12,7 @@ metadata:
   {{- end }}
 type: Opaque
 data:
+  {{- include "rabbitmq.validateValues" . }}
```

Comment:
Could you move this to the NOTES.txt file? This is where we normally put it in other charts.

Reply:
The thing is, that in the NOTES.txt this check is already in place, but `helm` go through the files in alphabetical order, so it fails on the `config-secret.yaml` before the validation in NOTES.txt file

As an option, the check might be moved to `templates/_validation.yaml` from NOTES.txt, keeping the validations in 1 place and doing them before the rendering. wdyt, @javsalgar ?

Reply:
That's interesting, I didn't know that. I'm fine by moving it to _validation.yaml

Reply:
I played with the naming, and it did not work with the underscore prefixed name, but it worked as expected with a normal name and helm does not clutter the output when there are no errors. Got TIL with it, hope you like this change too